Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/submission/ui/SubmissionTestFragmentViewModel.kt

```diff
@@ -75,6 +79,20 @@ class SubmissionTestFragmentViewModel @AssistedInject constructor(
 
     val tekHistory = MutableLiveData<List<TEKHistoryItem>>()
 
+    fun submit(checkDeviceTime: Boolean) = launch {
+        try {
+            srsSubmissionRepository.submit(SrsSubmissionType.SRS_REGISTERED_RAT, checkDeviceTime = checkDeviceTime)
```

Comment:
Should this default to SRS_REGISTERED_RAT?

Reply:
it is test menu, any type is fine